libs/platform/src/lib/form/form-generator/form-generator.service.ts

```diff
@@ -51,8 +53,14 @@ export class FormGeneratorService implements OnDestroy {
         @Optional()
         @SkipSelf()
         @Inject(FORM_GENERATOR_ITEM_CONFIG)
-        private _defaultConfig: Partial<DynamicFormFieldItem>
-    ) {}
+        private _defaultItemConfig: Partial<DynamicFormFieldItem>,
+        @Optional()
+        @SkipSelf()
+        @Inject(FORM_GENERATOR_CONFIG)
+        private _providedConfig: Partial<FormGeneratorConfig>
+    ) {
+        this._config = Object.assign({ ...this._config }, { ...this._providedConfig });
```

Comment:
Some inner props may be omitted, when config is created using `Object.assign`. Maybe it would be better to use lodash's merge, which is recursive? Also please note that config can be null, right?

Reply:
@dmitry-stepanenko config cannot be null since we use default provider in form generator module, and override it with `withConfig` method, which will not accept null, but can accept empty object which shouldn't break anything